firestore/runtime/src/main/java/io/quarkiverse/googlecloudservices/firestore/runtime/FirestoreProducer.java

```diff
@@ -1,19 +1,21 @@
 package io.quarkiverse.googlecloudservices.firestore.runtime;
 
-import java.io.IOException;
+import com.google.api.gax.retrying.RetrySettings;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+import io.quarkiverse.googlecloudservices.common.GcpBootstrapConfiguration;
+import io.quarkiverse.googlecloudservices.common.GcpConfigHolder;
+import io.quarkiverse.googlecloudservices.firestore.runtime.config.FirestoreConfiguration;
+import io.quarkiverse.googlecloudservices.firestore.runtime.config.FirestoreRetryConfiguration;
+import org.threeten.bp.Duration;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.inject.Default;
 import javax.enterprise.inject.Produces;
 import javax.inject.Inject;
 import javax.inject.Singleton;
-
-import com.google.auth.oauth2.GoogleCredentials;
-import com.google.cloud.firestore.Firestore;
-import com.google.cloud.firestore.FirestoreOptions;
-
-import io.quarkiverse.googlecloudservices.common.GcpBootstrapConfiguration;
-import io.quarkiverse.googlecloudservices.common.GcpConfigHolder;
+import java.io.IOException;
```

Comment:
These package changes didn't seems OK, please check that your IDE setting is OK, see https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style

You can use the following commands to format the code: `./mvnw process-sources`.

Reply:
My bad, I was looking for info about formatter but did not know to look in main Quarkus repository.